misc-utils/lsfd-unkn.c

```diff
@@ -1240,6 +1300,35 @@ static void anon_bpf_map_free(struct unkn *unkn)
 	free(data);
 }
 
+static void anon_bpf_map_get_more_info(struct anon_bpf_map_data *map_data)
+{
+	union bpf_attr attr = {
+		.map_id = (int32_t)map_data->id,
+		.next_id = 0,
+		.open_flags = 0,
+	};
+	struct bpf_map_info __attribute__((aligned(sizeof(uint64_t)))) info;
+	union bpf_attr info_attr = {
+		.info.info_len = sizeof(info),
+		.info.info = (__aligned_u64)&info,
+	};
+
+	int bpf_fd = syscall(SYS_bpf, BPF_MAP_GET_FD_BY_ID, &attr, sizeof(attr));
+	if (bpf_fd < 0)
+		return;
+
+	memset(&info, 0, sizeof(info));
```

Comment:
Can be folded into the definition: `info = { 0 };`

Reply:
Thank you. This is the 87th time I realized that I don't understand C.

Reply:
Updated.